test/unittest/transport/SharedMemTests.cpp

```diff
@@ -364,8 +366,9 @@ TEST_F(SHMRingBuffer, listeners_register_unregister)
     listener2->pop();
 }
 
-TEST_P(SHMRingBufferMultiThread, multiple_writers_listeners)
-{
+/*
+   TEST_P(SHMRingBufferMultiThread, multiple_writers_listeners)
```

Comment:
Should this test be left commented, fixed, or removed?

Reply:
Removed in c11f830d7